lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
Well... `mGet` is probably one of the only API route capable of generating such large responses, because there is currently no check on how many documents can be retrieved using it.
All other API routes I know of are either intrinsically limited in size, or paginated. The only exception I see are controllers plugin routes, where your proposal would be beneficial.

And the response size of a `mGet` call depends directly on the number of documents to fetch. 

Another argument I have is API harmonization. I agree, the `mGet` implementation benefits from a dedicated ES API route, but I think that's circumstantial. I would find it weird if all multi-documents routes  had to comply to this option, either in the `document` or in the `security` controllers, except for this one route.